tests/integration/api/test_extra_var.py

```diff
@@ -56,3 +56,15 @@ def test_retrieve_extra_var(client: APIClient):
 def test_retrieve_extra_var_not_exist(client: APIClient):
     response = client.get(f"{api_url_v1}/extra-vars/42/")
     assert response.status_code == status.HTTP_404_NOT_FOUND
+
+
+@pytest.mark.parametrize(
+    "extra_var", ["John", "John, ", "[John, 3,]", '{"name": "John" - 2 }']
```

Comment:
@Dostonbek1 Can you try these successful tests with both json and yaml strings.
e.g.
```
import yaml
import json
data = {'a': 1, 'b': 2}
json.dumps(data)
yaml.dump(data)
```